src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -35,7 +43,7 @@ type Scenario = {
   /**
    * SQL to teardown pre-test condition.
    */
-  down: string
+  down?: string
```

Comment:
Now that integration tests don't share state between themselves (same db), we don't really care about data teardown. Still, optional.